app/assets/javascripts/geoblacklight/templates/index_map_download.hbs

```diff
@@ -0,0 +1,19 @@
+{{#if downloadUrl}}
+  <li class="list-group-item download js-index-map-feature">
+    <h3 class="card-subtitle">Selected index map feature</h3>
```

Comment:
I think that this might be better to just have this say "Selected feature:" since we're already on an index map I don't think we need to have that in the label.

Reply:
Agreed and done. Thanks!